Sources/Sentry/include/SentryMeta.h

```diff
@@ -14,6 +15,28 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, class, copy) NSString *sdkName;
 
+/**
+ * Return an array of Serialized SDK packages
+ */
++ (NSArray<NSDictionary<NSString *, NSString *> *> *)getSdkPackagesSerialized;
```

Comment:
`m`: I don't think the SDK packages really belong here. The SentryMeta is only about version and SDK name. `getSdkPackagesSerialized` is a sign for me that it doesn't belong here. Instead, I would create an extra class just for the SDK packages. You can even write it in Swift.